packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -1052,6 +1052,395 @@ describe('RichTextEditor', () => {
+                        text: 'mailto:info@example.com'
+                    },
+                    {
+                        name: 'Anchor with mailto in href and email in text content',
+                        input: '<a href="mailto:info@example.com">info@example.com</a>',
+                        text: 'info@example.com'
+                    },
+                    {
+                        name: 'Anchor with invalid link',
+                        input: '<a href="javascript:vbscript:alert("not alert")">Invalid link</a>',
+                        text: 'Invalid link'
+                    },
+                    {
+                        name: 'Anchor with invalid link',
+                        input: '<a href="file:///path/to/local/file.txt">Invalid link</a>',
+                        text: 'Invalid link'
+                    },
+                    {
+                        name: 'Anchor with invalid link',
+                        input: '<a href="data:image/png;base64,iVBORw0KG...">Invalid link</a>',
+                        text: 'Invalid link'
+                    },
+                    {
+                        name: 'Anchor with invalid link',
+                        input: '<a href="tel:+1234567890">Invalid link</a>',
+                        text: 'Invalid link'
+                    },
+                    {
+                        name: 'Anchor with invalid link',
+                        input: '<a href="ssh://username@example.com">Invalid link</a>',
+                        text: 'Invalid link'
+                    },
+                    {
+                        name: 'Anchor with invalid link',
+                        input: '<a href="urn:isbn:0451450523">Invalid link</a>',
+                        text: 'Invalid link'
+                    },
+                    {
+                        name: 'Anchor with script tag',
+                        input: '<a href="#"><script>alert("attack")</script></a>',
+                        text: 'alert("attack")'
+                    }
+                ] as const;
+
+                parameterizeNamedList(
+                    differentProtocolLinks,
+                    (spec, name, value) => {
+                        spec(`${name} renders as plain text in editor`, () => {
+                            pageObject.pasteHTMLToEditor(value.input);
+
+                            expect(pageObject.getEditorTagNames()).toEqual([
+                                'P'
+                            ]);
+                            expect(pageObject.getEditorLeafContents()).toEqual([
+                                value.text
+                            ]);
+                        });
+                    }
+                );
+            });
+
+            describe('pasting various not supported nimble-anchor links should render as plain text', () => {
+                const differentProtocolLinks = [
```

Comment:
No need to separate these test cases from the set above. The test function is exactly the same. But I'd also question (again) what is really being tested here with the nested links, and if all of these cases are necessary.

Reply:
I have mentioned the reason for a nested link in the other comment and removed this repeated test case for the nimble-anchor case

Reply:
I have removed the `nimble-anchor` cases from the test files.